docs/UserGuide.md

```diff
@@ -387,6 +387,16 @@ To exit:
 
 This section is the table of all the commands available and their respective usage example in GradPad.
 
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes about the Command summary:**<br>
+
+* Words in curly braces (e.g. `{module code}`) refers to **compulsory** input that is required for the command to function.
+
+* Words in square brackets (e.g. `[c/module code]`) refers to **optional** input that can be entered.
+
+</div>
+
 Action | Format | Example
 --------|-------|----------
 **Add** | `add {module code}` | `add CS2100`
```

Comment:
Actually should we include the optional tags in this summary?

Reply:
I think its better to include though, because it is more clear for the reader else they would not know it is optional and the curly braces and square brackets were only used for the command summary so there shouldn't be inconsistency issues with regards to this note.

Reply:
Or are you suggesting that we remove the curly braces and square brackets entirely and replace with something else to indicate the optional fields?

Reply:
I agree with your first comment; I was thinking along the lines of this:
```suggestion
**Add** | `add {module code} [t/tags]` | `add CS2100 t/core`
```